project.clj

```diff
@@ -22,13 +22,9 @@
   :dependencies [[org.clojure/clojure "1.10.1"]
                  [clj-commons/clj-yaml "0.7.1"]
                  [metosin/jsonista "0.2.5"]
-                 [com.squareup.okhttp3/okhttp "4.5.0"]
-                 [com.github.jnr/jnr-unixsocket "0.30"]]
+                 [unixsocket-http "1.0.5"]]
   :plugins [[lein-ancient "0.6.15"]]
   :global-vars {*warn-on-reflection* true}
-  :aot [clj-docker-client.socket.TunnelingUnixSocket
-        clj-docker-client.socket.UnixDomainSocketFactory]
-  :resource-paths ["resources"]
```

Comment:
We need the resources to pack the swagger YAMLs before deploying, I think we need need this as the default is something else?

Reply:
Default should indeed be `resources`:
https://github.com/technomancy/leiningen/blob/master/leiningen-core/src/leiningen/core/project.clj#L221

Reply:
Right, I wasnt sure if its resources or src/clj_docker_client/resources. good to know 😄